tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -0,0 +1,288 @@
+          >CSV</a> <a
+            download="run_[[_runToDownload]]-tag-[[tag]].json"
+            href="[[_jsonUrl(_runToDownload)]]"
+          >JSON</a>
+          </div>
+        </div>
+      </template>
+    </div>
+  </tf-scalar-chart>
+  <style>
+    :host {
+      height: 235px;
+      width: 330px;
+      margin: 5px;
+      display: block;
+    }
+
+    :host[_expanded] {
+      height: 400px;
+      width: 100%;
+    }
+
+    paper-icon-button {
+      color: #2196F3;
+      border-radius: 100%;
+      width: 32px;
+      height: 32px;
+      padding: 4px;
+    }
+
+    paper-icon-button[selected] {
+      background: var(--tb-ui-light-accent);
+    }
+
+    .download-links {
+      display: flex;
+      height: 32px;
+    }
+
+    .download-links a {
+      font-size: 10px;
+      align-self: center;
+      margin: 2px;
+    }
+
+    .download-links paper-dropdown-menu {
+      width: 100px;
+      --paper-input-container-label: {
+        font-size: 10px;
+      }
+      --paper-input-container-input: {
+        font-size: 10px;
+      }
+    }
+  </style>
+</template>
+<script>
+  import * as ChartHelpers from '../vz-line-chart/vz-chart-helpers.js';
+  import {getRouter} from '../tf-backend/router.js';
+
+  /** 
+   * Allows:
+   * - "step" - Linear scale using the "step" property of the datum.
+   * - "wall_time" - Temporal scale using the "wall_time" property of the
+   * datum.
+   * - "relative" - Temporal scale using the "relative" property of the
+   * datum if it is present or calculating from "wall_time" if it isn't.
+   * @enum {string}
+   * */
+  
+  const X_TYPE = {
+    STEP: 'step',
+    RELATIVE: 'relative',
+    WALL: 'wall',
+  };
+
+  Polymer({
+      is: 'tf-scalar-card',
+      properties: {
+        runs: Array,  // of String
+        tag: String,
+
+        /**
+         * @type {X_TYPE}
+         */
+        xType: String,
+
+        active: Boolean,
+        ignoreYOutliers: Boolean,
+        requestManager: Object,
+        showDownloadLinks: Boolean,
+        smoothingEnabled: Boolean,
+        smoothingWeight: Number,
+        tagMetadata: Object,
+        tooltipSortingMethod: String,
+
+        // This function is called when data is received from the backend.
+        _processData: {
+          type: Object,
+          value: () => ((scalarChart, run, data) => {
+            const formattedData = data.map(datum => ({
+              wall_time: new Date(datum[0] * 1000),
+              step: datum[1],
+              scalar: datum[2],
+            }));
+            scalarChart.setSeriesData(run, formattedData);
+          }),
+          readOnly: true,
+        },
+
+        _expanded: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,  // for CSS
+        },
+
+        _logScaleActive: Boolean,
+
+        _dataUrl: {
+          type: Function,
+          value: () => getRouter().pluginRunTagRoute('scalars', '/scalars'),
```

Comment:
If you don't mind, I'd like to merge #494 and its dependency family before this; that family is currently halfway merged, blocking on review of #491, and it would be preferable to not introduce occurrences of old-style code right as we're trying to excise them.

Reply:
#491 LGTM

Reply:
Thanks to @dandelionmane for reviewing.